### ISSUE TITLE
Eventual Flatpak Runtime release?

### ISSUE THREAD

**Issue by fenglengshun** [Open-Wine-Components/ULWGL#9]
From what I see, this is intended to be a backend of sort, that other launchers can use.

In that case, for Flatpak, I'd imagine it would make sense for this to eventually be a runtime that the apps can add as a dependency. It would be more efficient than each apps redownloading ULWGL like they currently do with Wine.

Something like the [Proton-GE flathub runtime community build](https://github.com/flathub/com.valvesoftware.Steam.CompatibilityTool.Proton-GE) we had up to last year.

Might still be far away - since this looks fairly new - but I'd love if it's already part of the consideration from early on. Thanks regardless!

Edit - I just noticed, this is applied just directly in Proton-GE?

**Comment by GloriousEggroll** [comment]
ULWGL-Proton is now available
ULWGL-Launcher downloads and uses it as default if it does not find one. It places it in the standard compatibilitytools.d folder
ULWGL-Launcher basic flatpack yml has been created and should be testable if you want to build it locally

Please keep this in mind when testing:
https://github.com/ValveSoftware/steam-runtime/issues/529#issuecomment-1928286335

tldr: if you install from the build dir youll need to go allow the background permission manually in flatseal. if you install it from a local repo it should be fine

**Comment by GloriousEggroll** [comment]
latest rc has local flatpak repo tarballed for testing:
https://github.com/Open-Wine-Components/ULWGL-launcher/releases/tag/0.1-RC3

**Comment by GloriousEggroll** [comment]
closing. we are planning a flatpak release but there is no specific date and this bug report is not useful for "already planned" actions.
